app/services/atmosphere/appliance_vms_manager.rb

```diff
@@ -91,11 +91,20 @@ def instantiate_vm(tmpl, flavor, name)
     end
 
     def start_vm_on_cloud(tmpl, flavor, name)
+      nic = Atmosphere.nics[tmpl.compute_site.site_id]
+      if !(nic.blank?)
```

Comment:
`if nic` is not equivalent to `if !(nic.blank?)` (for example: an empty string passes `if nic` but does not satisfy `if !(nic.blank?)`.
Hound does not like `unless`.
Changed to `if nic.present?` instead.